docs/root/configuration/http/http_filters/_include/rate-limit-routes.yaml

```diff
@@ -0,0 +1,105 @@
+static_resources:
+  listeners:
+  - name: listener_0
+    address:
+      socket_address:
+        protocol: TCP
+        address: 0.0.0.0
+        port_value: 10000
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.http_connection_manager
+        typed_config:
+          "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+          stat_prefix: ingress_http
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: local_service
+              domains: ["*"]
+              routes:
+              - match:
+                  prefix: "/route0"
+                route:
+                  host_rewrite_literal: upstream.com
+                  cluster: upstream_com
+                  rate_limits:
+                  - actions:
+                    - source_cluster: {}
+                    - generic_key:
+                        descriptor_value: some_value0
```

Comment:
this requires a small change to associated docs

![image](https://github.com/user-attachments/assets/9d62c3e3-3367-49a7-966a-839085b7a2d3)